Modules/Loadable/Models/SubjectHierarchyPlugins/qSlicerSubjectHierarchyModelsPlugin.cxx

```diff
@@ -182,7 +183,7 @@ QString qSlicerSubjectHierarchyModelsPlugin::tooltip(vtkIdType itemID)const
   if (itemID == vtkMRMLSubjectHierarchyNode::INVALID_ITEM_ID)
   {
     qCritical() << Q_FUNC_INFO << ": Invalid input item";
-    return QString("Invalid");
+    return QString("Invalid item");
```

Comment:
All user-displayable strings must be translatable. Something like this will work, but test compilation (there are some requirements for the objects to get the `tr` function).

```suggestion
    return tr("Invalid item");
```